.github/workflows/build-wheels.yml

```diff
@@ -40,12 +40,12 @@ jobs:
         env:
           CIBW_ARCHS_LINUX: x86_64 aarch64
           CIBW_ARCHS_MACOS: x86_64 arm64
-          CIBW_ARCHS_WINDOWS: AMD64 x86 ARM64
-          CIBW_BEFORE_BUILD: pip install numpy scipy
+          CIBW_ARCHS_WINDOWS: AMD64 x86
```

Comment:
@piskvorky Sorry for some regression here. I would wait with ARM64 wheels since there are not many such systems, and also, Numpy, Scipy and other main libraries do not provide Windows arm64 wheels.

`oldest-supported-numpy` does not support that either. I would wait for them to start supporting arm64; otherwise, we must write a few ugly conditions for Numpy ourselves.